HackIllinois/ViewControllers/HIProfileViewController.swift

```diff
@@ -248,11 +254,21 @@ extension HIProfileViewController {
 
     func updateProfile() {
         guard let profile = HIApplicationStateController.shared.profile else { return }
+        guard let user = HIApplicationStateController.shared.user else { return }
         view.layoutIfNeeded()
 
         if let url = URL(string: profile.avatarUrl), let imgValue = HIConstants.PROFILE_IMAGES[url.absoluteString] {
             profilePictureView.changeImage(newImage: imgValue)
         }
```

Comment:
Comment out (or remove) any lines that refer to profilePictureView, since it is unused